.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,11 @@
+**Is your feature request related to a problem?**
+A new feature has been added.
+
+**What solution would you like?**
+Document the usage of the new feature.
+
+**What alternatives have you considered?**
+N/A
+
+**Do you have any additional context?**
+See please 
```

Comment:
Not sure what the context should be for this, could you update?

Reply:
So when the workflow runs, the PR number will be appended here ([line](https://github.com/opensearch-project/security/pull/1857/files#diff-8bebfdfe13265f982aff2186fd07882f7ba8f31e30677969734c39a758d552b3R28)).

Reply:
We actually have a new format @hdhalter and I worked on together. 

--- 
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 


**Tell us about your request. For example, what feature or area does it affect? What versions? If it’s a change to existing documentation, which pages (URLs) are affected?**
 
**Do you have any related resources to help us get started (e.g., link to a related fixed issue, design doc, etc.)? Any particular POCs (Eng or PM) who have more information?**

**Is there anything else you'd like to add?**